### DISCUSSION TITLE
remix error

### DISCUSSION THREAD

**Discussion by krishno16** [Cyfrin/foundry-full-course-cu#2352] [General]
![Screenshot 2024-08-30 133052 remix](https://github.com/user-attachments/assets/a20f92d8-dfdd-4333-8b9f-cadc6e3a9491)
why am i getting this error please explain me

**Comment by Shiva-Sai-ssb** [comment]
Hello @krishno16 , you have a typo in your SPDX license identifier line. It should be `SPDX-License-Identifier: MIT` instead of `SPDX-License-Identitifier: MIT`. Correcting this will remove the warning.

**Reply by krishno16** [reply]
thank you

**Reply by Shiva-Sai-ssb** [reply]
You're welcome.

**Comment by EngrPips** [comment]
Interesting.

**Comment by krishno16** [comment]
![Screenshot 2024-08-30 201027 question](https://github.com/user-attachments/assets/1bc8b178-60a5-4314-85b7-cfc6df6dcecc)
is this correct method for no4 queston?

**Reply by EngrPips** [reply]
I think so.

**Reply by krishno16** [reply]
i can't understand why the >,< this shows in different color but in lecture it's different

**Reply by EngrPips** [reply]
If things work correctly, I wouldn't sweat over that.

**Comment by AboloreDev** [comment]
I can see an error in your SPDX Identifier.

How to fix this:
Automatically remix helps in writing out the first two lines of code. 
in your code just type "pragma solidity (your version ) and remix automatically fill it up

**Reply by krishno16** [reply]
Write an empty contract that contains a SPDX License Identifier and compiles with version 0.8.11 or 0.8.13.

i can't understand how to write the versions basically greater or equals to 0.8.11 to less or equals to 0.8.13 i mean to say in single line 
i just wrote pragma solidity >=0.8.11 =<0.8.13;

**Reply by EngrPips** [reply]
Interesting
